configdialog.cpp

```diff
@@ -614,13 +614,23 @@ void PaletteCustomizeDialog::delete_current_palette() {
 		return;
 	}
 
+	auto all_palettes = m_palette_repo.get_all_custom_palettes();
+	size_t index = 0;
+	if (all_palettes.size() > 1) {
+		for (size_t i = 0; i < all_palettes.size(); i++) {
+			if (all_palettes[i].name == m_current_palette->name) {
+				index = (i > 0 ? i - 1 : i + 1);
```

Comment:
I doubt this is the most elegant way to do this, but it's the firs thing I thought of.